doc/source/user/building.rst

```diff
@@ -142,8 +142,7 @@ The default order for the libraries are:
 2. BLIS
 3. OpenBLAS
 4. ATLAS
-5. Accelerate (MacOS)
-6. BLAS (NetLIB)
+5. BLAS (NetLIB)
```

Comment:
Is this change actually true any more? It's possible we need the line of distutils that changed the search order, or a way for numpy to override it.

Reply:
Grr. Leaving `accelerate` in `blas_order` and `lapack_order` (in `numpy/distutils/system_info`) means it will still be discovered. So not modifying `system_info` means this PR is not actually doing what it says it is doing. 

I guess we can do something ugly like monkey patch system_info to remove them.

Reply:
Might be possible to do something less ugly by just subclassing `lapack_opt_info`.

A compromise might be to change only the `*_order` variables, but leave the class intact in case anyone wants accelerate directly.

Reply:
done in 6cb887f